cli_tools/gce_vm_image_import/importer/disk_inspection_processor.go

```diff
@@ -41,8 +41,13 @@ func (p *diskInspectionProcessor) process(pd persistentDisk,
 		return pd, err
 	}
 
-	pd.isUEFICompatible = p.args.UefiCompatible || ir.HasEFIPartition
-	loggableBuilder.SetUEFIMetrics(pd.isUEFICompatible, ir.HasEFIPartition)
+	isHybrid := ir.UEFIBootable && ir.BIOSBootable
+	if !p.args.UefiCompatible && isHybrid {
+		log.Printf("This disk can boot with both BIOS and UEFI. It will be imported as BIOS booting. " +
+			"If you want to use UEFI booting, please specify arg '--uefi_compatible'.")
```

Comment:
This flag is currently not available in gcloud and probably it will be called -uefi-compatible (since we don't use underscores in gcloud flags). Might be confusing before we make this flag available in gcloud unless users are using API. 

It's probably a better idea to link to https://cloud.google.com/compute/docs/import/importing-virtual-disks#importing_a_virtual_disk_with_uefi_bootloader

It already documents how to achieve this with API. Once gcloud flag is available, docs will be updated.